web-gateway/app/controllers/ItemForm.java

```diff
@@ -31,30 +31,21 @@
         List<ValidationError> errors = new ArrayList<>();
         
         try {
-            Currency c = Currency.valueOf(currency);
-
+        	Currency c = Currency.valueOf(currency);
```

Comment:
@yg-apaza this answer on Stack Overflow covers it well http://stackoverflow.com/a/408449/29470